src/drivers/roboclaw/RoboClaw.cpp

```diff
@@ -174,7 +174,7 @@ void RoboClaw::taskMain()
 
 	while (!taskShouldExit) {
 
-		int pret = poll(fds, sizeof(fds) / sizeof(pollfd), waitTime / 1000);
+		int pret = px4_poll(fds, sizeof(fds) / sizeof(px4_pollfd_struct_t), waitTime / 1000);
```

Comment:
I think px4_poll was removed above, is this dead code or something ?

Reply:
no; previously px4_poll was a mix of polling orbs and files. In almost all cases it is polling uorbs, so I changed px4_poll into orb_poll, which is implemented then in uORB. I kept the same semantics for orb_poll to minimize code changes all around the px4, although it would have been cleaner to re-design this interface to something not similar to "poll".